types/event_record.go

```diff
@@ -114,6 +115,27 @@ type EventImOnlineHeartbeatReceived struct {
 	Topics      []Hash
 }
 
+type Exposure struct {
```

Comment:
Can you add a comment to the exported structs pls

Reply:
Done